### PR TITLE
logger: Use hostname as cloudwatch stream

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: internal/logger/logger.go [hunk 1]
```diff
@@ -114,10 +104,12 @@ func NewLogger(level string, key *string, secret *string, region *string, group
 	}
 
 	if key != nil {
-		log.SetFormatter(NewCloudwatchFormatter())
```
Comment (msehnout) [inline internal/logger/logger.go:117]: Why was this line removed?

Comment (Gundersanne) [reply internal/logger/logger.go:117]: it moved to line 108. Just assigning the formatter to a variable since it's used on line 112 as well.

**Review by msehnout — APPROVED** [review]
Looks good to me! Thanks for figuring this out, hope it helps :-)
